### DISCUSSION TITLE
Microcks API endpoint Response getting cut

### DISCUSSION THREAD

**Discussion by cvsudheer108** [microcks/microcks#918] [Q&A] [answered]
Hello,
I have configured Microcks locally and imported a postman collection successfully that predefines the responses.
I am able to get the responses, but find that the responses are getting cut as shown below.. I couldn't find any setting. For different endpoints, it is getting cust at different lengths. 

<img width="778" alt="Screenshot 2023-09-20 at 8 31 08 AM" src="https://github.com/microcks/microcks/assets/22774621/ff66d466-7bf2-4f78-b7e0-b2244424994e">

For another endpoint, the response is much bigger, but getting cut at some point.

<img width="707" alt="Screenshot 2023-09-20 at 8 37 39 AM" src="https://github.com/microcks/microcks/assets/22774621/1c213648-4591-48b8-924f-0eefd2417316">

Has anybody come across this issue? Please let me know

**Comment by lbroudoux** [comment]
Hello! We didn't face this issue AFAIK...
Have you checked if your Postman Collection example may contain `Content-Length` headers on response with values that are not accurate? Could you share a small reproducer for further investigation?

**Reply by cvsudheer108** [reply]
Sure.. I am sending a sample api with the responses configured in postman. Just import that in to Microcks and see tge 200, 400 or 503 responses. All are getting cut at different lengths.
Since it doesn't allow JSON files to be attached, I have renamed it to .txt file. Please rename it back to .json before importing.




BTW I confirmed that I am not sending any Content-lengthe in the header.
Thank you!

**Reply by lbroudoux** [reply, marked as answer]
Hey @cvsudheer108!

I've checked it out, and you actually have `Content-Length' header in the response that is inaccurate. On the `200` sample, for example, the `Content-Length` in response is set to `12398`, but the response payload is actually `19168`. This makes your HTTP client truncate the response on the client-side. You have to adjust them or remove them from your collection.

**Reply by cvsudheer108** [reply]
Thank you! will see how that wrong content lenght is getting set.
